portfolio/src/main/java/com/google/sps/quote/Quote.java

```diff
@@ -0,0 +1,22 @@
+package com.google.sps.quote;
+
+/**
+ * A quote consist of the author and quote text
+ */
```

Comment:
nit: A better comment style would be "Class that stores information about a Quote including author and text"